script/Amex-Select-Offers.js

```diff
@@ -40,7 +40,11 @@
   }
 
   function inProgressButton() {
+    button.removeEventListener("mouseover", handleMouseOver);
+    button.removeEventListener("mouseout", handleMouseOut);

```

Comment:
these are not needed, if the listener is removed here no need to remove once `disableButton()` is called, so this is redundant.

Reply:
Not needed for sure. I had copy and pasted from other.